tests/unit/all.ts

```diff
@@ -1 +1 @@
-export var removeThis = 1;
+import 'dojo/has!host-browser?./dom:';
```

Comment:
As per comment on https://github.com/bitpshr/dom/commit/dddf44e44a354bd0b3cf498b464529abc599e66f, Bryan and I think that since this package is known to be browser-only, checking for `host-browser` all the time is just superfluous extra work that can be removed.